lib/stringutil.gr

```diff
@@ -1,50 +1,50 @@
 import String from "string"
 import Array from "array"
 
-// Return a String that is the reverse of the given String.
-//
-// @param str: The string to reverse.
-// @return String A reversed version of the given string
+/**
+ * Return a String that is the reverse of the given String.
+ *
+ * @param str: The string to reverse.
+ * @returns A reversed version of the given string
+ */
 export let reverse = (str: String) => {
-    let chars = String.explode(str)
-    let clen = Array.length(chars)
-    let rev = Array.init(clen, (index) => {
-        let last = clen - index - 1
-        chars[last]
-    })
-    String.implode(rev)
+  let chars = String.explode(str)
+  let rev = Array.reverse(chars)
```

Comment:
This makes me happy... and also makes me feel slightly guilty because I had intended to contribute a `String.reverse`. 😁

Reply:
Not to worry, you still have `Array.sort` and `List.sort`.
https://github.com/grain-lang/grain/issues/125